src/ir_Fujitsu.cpp

```diff
@@ -0,0 +1,218 @@
+// Copyright 2017 Jonny Graham
+#include "ir_Fujitsu.h"
+#include <algorithm>
+#include "IRrecv.h"
```

Comment:
Not sure, but I don't think you need this header as you don't have a decoder (yet).